scan_filter_chain:
- name: range
  type: laser_filters/LaserScanRangeFilter
  params:
    use_message_range_limits: false
    lower_threshold: 0.1
    upper_threshold: 40
    lower_replacement_value: -.inf
    upper_replacement_value: .inf